src/app/components/search/DateRangeFilter.js

```diff
@@ -290,6 +291,30 @@ const DateRangeFilter = ({
 
   const getEndDateStringOrNull = () => getDateStringOrNull('end_time');
 
+  // Due to the need for ISO datetime string in the Datepicker textfield component
+  // need to take the ISO value and break into substring and calculate UTC from that.
+  const getUTCOffset = (isoDateTimeString, isStart) => {
+    let [hours, minutes] = isoDateTimeString.split('T')[1].split('.')[0].split(':').map(Number);
+
+    if (!isStart) { // removed 59 minutes when added for end time
+      minutes -= 59;
+    }
+
+    let tzSign;
+    if (hours > 12) { // these are positive UTC value zones
+      tzSign = '+';
+      hours = 24 - hours;
+      hours -= isStart ? 0 : 1; // used for rounding up 59 minutes to 1 hr
+      hours -= isStart && minutes !== 0 ? 1 : 0; // special case for non-00 timezones
+    } else {
+      tzSign = '-';
+      hours += isStart ? 0 : 1;
+      hours += isStart && minutes !== 0 ? 1 : 0;
+    }
+    const tzOffsetMinutes = String(Math.abs(minutes)).padStart(2, '0');
```

Comment:
Wow, I didn't know about `padStart`!

Reply:
Added a few unit tests using different ISO strings created in different zones using the Chrome DevTools Sensors feature.  Date will always using the current timezone of the user so fudging it in the browser is the only way to get it to spit out the correct ISO string.